department-of-veterans-affairs/va.gov-cms
# VACMS-2509: Remove old blobby content model for Facility health services

## Description

closes #2509 
~Blocked by https://github.com/department-of-veterans-affairs/content-build/pull/1031~
Blocked by https://github.com/department-of-veterans-affairs/content-build/pull/1032   This can not be merged until https://github.com/department-of-veterans-affairs/content-build/pull/1032#pullrequestreview-907821757 is merged AND makes its way into CMS codebase.

- Removed "field_body" field from VAMC Facility health service content type.
- Updated behat feature.
- Added hook_update_n in va_gov_vamc module.

## Testing done
Passes all tests

## Screenshots
![VACMS-2509-hook_update_N-returned-message](https://user-images.githubusercontent.com/846871/157829426-4270d64a-8391-4982-969b-2513058db36f.png)

![VACMS-2509-update php-returned-message](https://user-images.githubusercontent.com/846871/157829449-52b249fc-82a3-40f4-8646-fb1f9a3ecb9d.png)

## QA steps
As user administrator
1. Go to `/admin/structure/types/manage/health_care_local_health_service/fields`
   - [x] Verify and confirm the `field_body` (label: **Facility description of service**) is not part of the listed fields
2. Go to VAMC System Banner Alert with Situation Updates: `/admin/structure/types/manage/full_width_banner_alert/fields`
    - [x] Verify and confirm the `field_body` (label: **Alert body**) is listed in the available fields
3. Go to VAMC System Health Service: `/admin/structure/types/manage/regional_health_care_service_des/fields`
    - [x] Verify and confirm the `field_body` (label: **VAMC system service description**) is listed in the available fields
4. Verify and confirm that hook_update_N returned a successful message:
    - [x] In a terminal run `ddev drush updb` and check that this message is printed out: "_The field field_body have been removed from the VAMC Facility Health Service content type._"
    - [x] Run `update.php` (either by going to `/update.php` or run it from the toolbar menu) and check that this message is printed out: "_The field field_body have been removed from the VAMC Facility Health Service content type._"

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
